common/schemes/wc_stasis_trap_scheme.txt

```diff
@@ -26,9 +26,6 @@
 	
 	valid = {
 		valid_hostile_scheme_target_trigger = yes
-		scope:target = {
-			is_in_stun_trigger = no
-		}
```

Comment:
The comment is so wrong. If you have it in the interaction, it doesn't mean you don't need it here.

Why? Because it made the scheme invalid if someone else stunned this character.